src/chrome/content/rules/UPS.com.xml

```diff
@@ -22,28 +22,42 @@
 <ruleset name="UPS.com (partial)">
 
 	<target host="ups.com" />
-	<target host="*.ups.com" />
+		<exclusion pattern="^http://ups.com/$" />
+	<target host="www.ups.com" />
+		<exclusion pattern="^http://www.ups.com/$" />
+	<target host="wwwapps.ups.com" />
+	<target host="www.apps.ups.com" />
+	<target host="campusship.ups.com" />
+	<target host="www.campusship.ups.com" />
+
 		<!--
 			Redirects to http:
 						-->
-		<!--exclusion pattern="^http://www\.ups\.com/content/globalentry$" /-->
-		<!--
-			all ur pkgs r now belong to nsa
+		<exclusion pattern="^http://www\.ups\.com/content/globalentry$" />
+			<test url="http://www.ups.com/content/globalentry" />
 
-			(Now redirects to login:
-						-->
-		<!--exclusion pattern="^http://www\.ups\.com/WebTracking/track($|\?)" /-->
 		<!--
-			Exceptions:
-					-->
-		<exclusion pattern="^http://www\.ups\.com/+(?!assets/|favicon\.ico|img/|styles/|stylesheets/)" />
-
+			Now redirects to login:
+						-->
+		<exclusion pattern="^http://www\.ups\.com/WebTracking/track($|\?)" />
+			<test url="http://www.ups.com/WebTracking/track" />
+			<test url="http://www.ups.com/WebTracking/track?" />
```

Comment:
https://wwwapps.ups.com/WebTracking/track
HTTP/1.1 302 Found
Date: Wed, 25 Feb 2015 07:46:51 GMT
Server: Apache
X-Frame-Options: SAMEORIGIN
Location: https://www.ups.com/one-to-one/login?returnto=https%3a//wwwapps.ups.com/WebTracking/track&reasonCode=-1&appid=ETRACK